lib/cxml/item_detail.rb

```diff
@@ -35,8 +37,12 @@ def render(node)
         node.ManufacturerPartID(manufacturer_part_id) if manufacturer_part_id.present?
         node.ManufacturerName(manufacturer_name) if manufacturer_name.present?
         node.LeadTime(lead_time) if lead_time.present?
+        if extrinsics.present?
```

Comment:
Could use the &. operator for this

@extrinsics&.each do |extrinsic_name, extrinsic_value|
  node.Extrinsic(extrinsic_value, 'name' => extrinsic_name)
end